Makefile

```diff
@@ -25,16 +26,22 @@ diff/chameneos_redux.diff: out/chameneos_redux.txt ref/chameneos_redux.txt
 
 bin/binary_trees: src/binary_trees.rs lib/$(ARENA).pkg
 
+bin/fasta: src/fasta.rs lib/$(NUM_CPU).pkg
+
+bin/fasta_redux: src/fasta_redux.rs lib/$(NUM_CPU).pkg
+
 bin/regex_dna: src/regex_dna.rs lib/$(REGEX).pkg
 
 lib/%.pkg:
-	mkdir -p tmp
-	rm -rf tmp/$(call crate,$*)-deps
-	cargo new tmp/$(call crate,$*)-deps
-	echo '\n[dependencies]\n$(call crate,$*) = "$(call version,$*)"' >> tmp/$(call crate,$*)-deps/Cargo.toml
-	cargo build --release --manifest-path tmp/$(call crate,$*)-deps/Cargo.toml
-	mkdir -p lib
-	cp tmp/$(call crate,$*)-deps/target/release/deps/* lib/
+	@mkdir -p tmp
+	@rm -rf tmp/$(call crate,$*)-deps
+	@cargo new tmp/$(call crate,$*)-deps
+	@echo >> tmp/$(call crate,$*)-deps/Cargo.toml
+	@echo '[dependencies]' >> tmp/$(call crate,$*)-deps/Cargo.toml
```

Comment:
You can use printf instead of echo. It is the most portable program to print on the shell, and it's much more powerful than echo.